codebuild/scripts/release_tag.sh

```diff
@@ -18,10 +18,10 @@ function retag_image()
   docker push $image:$new_tag
 }
 
-CODEBUILD_GIT_TAG="$(git describe --tags --exact-match 2>/dev/null)"
+GIT_TAG="$(git describe --tags --exact-match 2>/dev/null)"
```

Comment:
Will we have more than one tag now? e.g. v1.1 -> v1.1.6 -> commit sha?

Reply:
In git we will only have `vX.Y.Z` tags, but in the containers and artifacts we will have each of the types.

Reply:
Ah, I see (although I think you mean that git will only have `vX.Y` tags?)

Reply:
Git will only have `rX.Y` branches, but all tags and releases will be `vX.Y.Z`